packages/ai-ide/src/browser/ai-configuration/model-aliases-configuration-widget.tsx

```diff
@@ -0,0 +1,272 @@
+        }
+        this.matchingAgentIdsForAliasMap = aliasMap;
+    }
+
+    protected handleAliasSelectedModelIdChange = (alias: LanguageModelAlias, event: React.ChangeEvent<HTMLSelectElement>): void => {
+        const newModelId = event.target.value || undefined;
+        const updatedAlias: LanguageModelAlias = {
+            ...alias,
+            selectedModelId: newModelId
+        };
+        this.languageModelAliasRegistry.ready.then(() => {
+            this.languageModelAliasRegistry.addAlias(updatedAlias);
+        });
+    };
+
+    render(): React.ReactNode {
+        const selectedAliasId = this.aiConfigurationSelectionService.getSelectedAliasId();
+        const selectedAlias = this.aliases.find(alias => alias.id === selectedAliasId);
+        return (
+            <div className="model-alias-configuration-main">
+                <div className="model-alias-configuration-list preferences-tree-widget theia-TreeContainer ai-model-alias-list">
+                    <ul>
+                        {this.aliases.map(alias => (
+                            <li
+                                key={alias.id}
+                                className={`theia-TreeNode theia-CompositeTreeNode${alias.id === selectedAliasId ? ' theia-mod-selected' : ''}`}
+                                onClick={() => this.aiConfigurationSelectionService.setSelectedAliasId(alias.id)}
+                            >
+                                <span>{alias.id}</span>
+                            </li>
+                        ))}
+                    </ul>
+                </div>
+                <div className="model-alias-configuration-panel preferences-editor-widget">
+                    {selectedAlias ? this.renderAliasDetail(selectedAlias, this.languageModels) : (
+                        <div>
+                            {nls.localize('theia/ai/core/modelAliasesConfiguration/selectAlias', 'Please select a Model Alias.')}
+                        </div>
+                    )}
+                </div>
+            </div>
+        );
+    }
+
+    protected renderAliasDetail(alias: LanguageModelAlias, languageModels: LanguageModel[]): React.ReactNode {
+        const agentIds = this.matchingAgentIdsForAliasMap.get(alias.id) || [];
+        const agents = this.agentService.getAllAgents().filter(agent => agentIds.includes(agent.id));
+        const resolvedModel = this.resolvedModelForAlias.get(alias.id);
+        return (
+            <div>
+                <div className="settings-section-title settings-section-category-title ai-alias-detail-title">
+                    <span>{alias.id}</span>
+                </div>
+                {alias.description && <div className="ai-alias-detail-description">{alias.description}</div>}
+                <div className="ai-alias-detail-selected-model">
+                    <label>{nls.localize('theia/ai/core/modelAliasesConfiguration/selectedModelId', 'Selected Model')}: </label>
```

Comment:
The UI / Evaluation does not properly handle the case that the user selected a model previously which is no longer available. The UI shows as if the "priority list" is selected

<img width="567" height="313" alt="image" src="https://github.com/user-attachments/assets/99c511e9-5b91-48a1-9f57-60f3d08d1b5e" />

Instead it should look like this

<img width="529" height="324" alt="image" src="https://github.com/user-attachments/assets/99a64183-5cf2-40a4-bd81-72654ca1fb5b" />